.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist/
+server/.env
```

Comment:
We should also add a root level `.env`

Reply:
For project level vars like those that might eventually effect firmware building, app building, external APIs, etc.

And I'm not suggesting we implement it, just thought that adding `.env` to the file would cover all potential cases rather than only `server/.env`. Just like the `node_modules` and `dist` above also ignore `{server,app}/node_modules` and `app/dist`.